tests/openwisp2/sample_radius/migrations/0028_alter_organizationradiussettings_allowed_mobile_prefixes_and_more.py

```diff
@@ -2,7 +2,8 @@
 
 from django.db import migrations
 
-import openwisp_radius.base.fields
+import openwisp_radius
```

Comment:
not sure what this is imported for

Reply:
It is imported for the validator of `password_reset_url` field. 

https://github.com/openwisp/openwisp-radius/blob/6e3dc35088f0ba3190f3b552860e92858c96e558/tests/openwisp2/sample_radius/migrations/0028_alter_organizationradiussettings_allowed_mobile_prefixes_and_more.py#L147-L162